diff --git a/node_modules/@matterlabs/signature-checker/contracts/SignatureChecker.sol b/node_modules/@matterlabs/signature-checker/contracts/SignatureChecker.sol
index 3d39fb0..bff22fc 100644
--- a/node_modules/@matterlabs/signature-checker/contracts/SignatureChecker.sol
+++ b/node_modules/@matterlabs/signature-checker/contracts/SignatureChecker.sol
@@ -1,7 +1,7 @@
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import "@openzeppelin/contracts/interfaces/IERC1271.sol";
+import "@openzeppelin/contracts-4.8/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts-4.8/interfaces/IERC1271.sol";
 
 library SignatureChecker {
     /** 
